server/chart/chart.test.js

```diff
@@ -0,0 +1,29 @@
+import { MongoMemoryServer } from "mongodb-memory-server";
+import mongoose from "mongoose";
+import request from "supertest";
+import app from "../app";
+import Chart from "./chart.model";
+
+describe("/api/chart tests", () => {
+  const mongod = new MongoMemoryServer();
+
+  beforeAll(async () => {
+    const uri = await mongod.getConnectionString();
+    await mongoose.connect(uri, { useNewUrlParser: true });
+  });
+
+  afterAll(async () => {
+    await mongoose.disconnect();
+    await mongod.stop();
+  });
+
+  afterEach(async () => {
+    await Chart.remove({});
+  });
+
+  it("should be able to create a new chart", async () => {
+    const dateNow = new Date();
+
+    const postResponse = await request(app).post("/api/chart").send({});
```

Comment:
what's being done with the response of this?
A test should always contain assertions. otherwise it will always pass regardless of failure. In this case, you should check the status of postResponse by accessing its status field. and maybe looking at the body and expecting something that represents a chart